chromadb/migrations/sysdb/00006-segments-fk.sqlite.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE segments_temp (
+                          id TEXT PRIMARY KEY,
+                          type TEXT NOT NULL,
+                          scope TEXT NOT NULL,
+                          topic TEXT,
+                          collection TEXT REFERENCES collections(id)
+);
+
+INSERT INTO segments_temp SELECT * FROM segments;
+
+DROP TABLE segments;
+
+ALTER TABLE segments_temp RENAME TO segments;
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line 14]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->Add disable fk file<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/chroma-core/chroma/1666?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->

Reply:
File added